manuscript/setup/dev.md

```diff
@@ -93,10 +94,16 @@ Follow this section ONLY if you're NOT planning to use EKS, AKS, or GKE
 
 ```bash
 # Install `yq` CLI from https://github.com/mikefarah/yq
+# 'yq' is a lightweight and portable command-line YAML processor
+
 # Install `jq` CLI from https://stedolan.github.io/jq/download
+# 'jp' is a lightweight and portable command-line JSON processor
 
 kubectl create namespace dev
 
+# Helm is a package manager for Kubernetes applications
```

Comment:
This is great!

Let me throw a random thought... How about creating a YT Short (<= 60 seconds) for those bitesize tips on your (Whitney's) channel?

Reply:
Let's taco bout it next call!